lib/galaxy/tools/repositories.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, app_name,
         self.temporary_path = tempfile.mkdtemp(prefix='tool_validation_')
         self.config.tool_data_table_config = os.path.join(self.temporary_path, 'tool_data_table_conf.xml')
         self.config.shed_tool_data_table_config = os.path.join(self.temporary_path, 'shed_tool_data_table_conf.xml')
+        self.config.interactivetools_enable = True
```

Comment:
That enabled TS upload, are we sure that the interface is stable now ?

Reply:
I think it would be reasonable to state so. There has been pretty decent uptake of it currently. I don't think there been altering changes in over a year. Would be good to hear other input though.

Are there any open issues or PRs that would not be compatible?